frontend/src/metabase-lib/tests/drills-pivot.unit.spec.ts

```diff
@@ -0,0 +1,339 @@
+      aggregation: {
+        column: createCountDatasetColumn(),
+        value: 10,
+      },
+      breakouts: [
+        {
+          column: createProductsCategoryDatasetColumn({
+            source: "breakout",
+          }),
+          value: "2020-01-01",
+        },
+      ],
+    });
+
+    it("should drill thru an aggregated cell without location columns", () => {
+      const { drill } = findDrillThru(
+        query,
+        stageIndex,
+        clickObject,
+        drillType,
+      );
+      const pivotTypes = Lib.pivotTypes(drill);
+      expect(pivotTypes).toEqual(["category", "time"]);
+      verifyDrillThru(query, stageIndex, drill, pivotTypes);
+    });
+  });
+
+  describe("1 aggregation, 1 numeric breakout", () => {
+    const query = createQueryWithClauses({
+      aggregations: [{ operatorName: "count" }],
+      breakouts: [{ columnName: "TOTAL", tableName: "ORDERS" }],
+    });
+    const clickObject = createAggregatedCellClickObject({
+      aggregation: {
+        column: createCountDatasetColumn(),
+        value: 10,
+      },
+      breakouts: [
+        {
+          column: createOrdersTotalDatasetColumn({
+            source: "breakout",
+          }),
+          value: 10,
+        },
+      ],
+    });
+
+    it("should not drill thru an aggregated cell", () => {
+      const drill = queryDrillThru(query, stageIndex, clickObject, drillType);
+      expect(drill).toBeNull();
+    });
+  });
+
+  describe("1 aggregation, 1 address breakout", () => {
+    const query = createQueryWithClauses({
+      aggregations: [{ operatorName: "count" }],
+      breakouts: [{ columnName: "STATE", tableName: "PEOPLE" }],
+    });
+    const clickObject = createAggregatedCellClickObject({
+      aggregation: {
+        column: createCountDatasetColumn(),
+        value: 10,
+      },
+      breakouts: [
+        {
+          column: createPeopleStateDatasetColumn({
+            source: "breakout",
+          }),
+          value: "2020-01-01",
+        },
+      ],
+    });
+
+    it("should drill thru an aggregated cell without location columns", () => {
+      const { drill } = findDrillThru(
+        query,
+        stageIndex,
+        clickObject,
+        drillType,
+      );
+      const pivotTypes = Lib.pivotTypes(drill);
+      expect(pivotTypes).toEqual(["category", "time"]);
+      verifyDrillThru(query, stageIndex, drill, pivotTypes);
+    });
+  });
+
+  describe("1 aggregation, 1 date and 1 category breakout", () => {
+    const query = createQueryWithClauses({
+      aggregations: [{ operatorName: "count" }],
+      breakouts: [
+        { columnName: "CREATED_AT", tableName: "ORDERS" },
+        { columnName: "CATEGORY", tableName: "PRODUCTS" },
+      ],
+    });
+    const clickObject = createAggregatedCellClickObject({
+      aggregation: {
+        column: createCountDatasetColumn(),
+        value: 10,
+      },
+      breakouts: [
+        {
+          column: createOrdersTotalDatasetColumn({
```

Comment:
Trying this with my pivots PR found a mistake here. Unfortunately I spent a while debugging the pivot code before realizing the test input was inconsistent.

The `breakouts` above want `CREATED_AT` and `CATEGORY` (and the test description calls from date and category breakouts so that seems right) but this is constructing the click object with a `TOTAL` column (and a value of `"2020-01-01"`). Changing this line to call `createOrdersCreatedAtDatasetColumn()` fixes it.